doc/docker/Dockerfile-solr

```diff
@@ -18,5 +20,10 @@ RUN mkdir -p /opt/solr/server/ez/template \
 # Set our core config as home
 ENV SOLR_HOME /opt/solr/server/ez
 
-# Make sure core is created on startup
-CMD ["solr-create", "-c", "collection1", "-d", "/opt/solr/server/ez/template"]
+# Make sure cores are created on startup
+RUN bin/solr -s ez -p "$SOLR_PORT" \
+ && bin/solr create_core -c collection1 -d server/ez/template -p "$SOLR_PORT" \
+ && bin/solr create_core -c econtent1 -d server/ez/template -p "$SOLR_PORT" \
+ && bin/solr create_core -c econtent2 -d server/ez/template -p "$SOLR_PORT" \
+ && wget -q -O /dev/null http://127.0.0.1:"$SOLR_PORT"/solr/admin/cores?action=RENAME\&core=econtent1\&other=econtent\&wt=json \
```

Comment:
Renaming cores is necessary according to this comment >> https://github.com/ezsystems/ezcommerce-advanced/commit/c5f23ec67013feca0e4a6539ffc2157ad4e33441